fhir-openapi/src/main/webapp/META-INF/openapi.json

```diff
@@ -195444,6 +195800,230 @@
                         ]
                     }
                 ]
+            },
+            "Parameters_Parameter": {
+                "allOf": [
+                    {
+                        "$ref": "#/components/schemas/BackboneElement"
+                    },
+                    {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "pattern": "[ \\r\\n\\t\\S]+",
+                                "description": "The name of the parameter (reference to the operation definition)."
+                            },
+                            "valueBase64Binary": {
+                                "type": "string",
+                                "pattern": "(\\s*([0-9a-zA-Z\\+/=]){4}\\s*)+",
+                                "description": "If the parameter is a data type."
```

Comment:
All the types have the same description "If the parameter is a data type."
Should it say something more specific for each one?

Reply:
This is pulled directly from the profiles-resources.json.  All these types have the same description:

        {
          "id" : "Parameters.parameter.value[x]",
          "path" : "Parameters.parameter.value[x]",
          "short" : "If parameter is a data type",
          "definition" : "If the parameter is a data type.",
          "min" : 0,
          "max" : "1",
          "base" : {
            "path" : "Parameters.parameter.value[x]",
            "min" : 0,
            "max" : "1"
          },
          "type" : [{
            "code" : "base64Binary"
          },
          {
            "code" : "boolean"
          },
          {
            "code" : "canonical"
          },
          {
            "code" : "code"
          },
          {
            "code" : "date"
          },
          {
            "code" : "dateTime"
          },
          {
            "code" : "decimal"
          },
          {
            "code" : "id"
          },
          {
            "code" : "instant"
          },
          {